### ISSUE TITLE
clear_exporting_parameters false positive: exporting parameter checked with IS SUPPLIED/IS REQUESTED

### ISSUE THREAD

**Issue by rubber-duck-53** [abaplint/abaplint#4187]
```abap
CLASS zcl_test_clear_exporting_param DEFINITION PUBLIC CREATE PUBLIC.
  PUBLIC SECTION.
    METHODS with_is_requested
      EXPORTING ev_result TYPE i.
    METHODS with_is_supplied
      EXPORTING ev_result TYPE i.
    METHODS direct_write
      EXPORTING ev_result TYPE i.
ENDCLASS.

CLASS zcl_test_clear_exporting_param IMPLEMENTATION.
  METHOD with_is_requested.
    IF ev_result IS REQUESTED.
      ev_result = 1.
    ENDIF.
  ENDMETHOD.

  METHOD with_is_supplied.
    IF ev_result IS SUPPLIED.
      ev_result = 1.
    ENDIF.
  ENDMETHOD.

  METHOD direct_write.
    ev_result = 1.
  ENDMETHOD.
ENDCLASS.
```

## Expected

No findings — Same behavior for all three methods

## Actual

```
zcl_test_clear_exporting_param.clas.abap[13, 8] - EXPORTING parameter "ev_result" read before it is cleared or assigned (clear_exporting_parameters) [E]
zcl_test_clear_exporting_param.clas.abap[19, 8] - EXPORTING parameter "ev_result" read before it is cleared or assigned (clear_exporting_parameters) [E]
```

**Comment by rubber-duck-53** [comment]
@larshp `GET REFERENCE OF` and `ASSIGN TO FIELD-SYMBOL` usages on exporting parameters might also be worth considering as false positives

**Comment by larshp** [comment]
@rubber-duck-53 try again, latest version

**Comment by rubber-duck-53** [comment]
Thanks, works as expected now.
